website/pages/en/developing/creating-a-subgraph/starting-your-subgraph.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Starting Your Subgraph
+---
+
+## Overview
+
+The Graph is home to thousands of subgraphs already available for query, so check [The Graph Explorer](https://thegraph.com/explorer) and find one that already matches your needs.
+
+When you create a [subgraph](/subgraphs/), you create a custom open API that extracts data from a blockchain, processes it, stores it, and makes it easy to query via GraphQL.
+
+Subgraph development ranges from simple scaffold subgraphs to advanced, specifically tailored subgraphs.
+
+### Start Building
+
+Start the process and build a subgraph that matches your needs:
+
+1. [Install the CLI](/developing/creating-a-subgraph/install-the-cli/)- Set up your infrastructure
+2. [Subgraph Manifest](/developing/creating-a-subgraph/subgraph-manifest/)- Understand a subgraph's key component
+3. [The Graph Ql Schema](/developing/creating-a-subgraph/ql-schema/)- Write your schema
+4. [Writing AssemblyScript Mappings](/developing/creating-a-subgraph/assemblyscript-mappings/)- Write your mappings
+5. [Advanced Features](/developing/creating-a-subgraph/advanced/)- Customize your subgraph with advanced features
```

Comment:
Really nice! Just a nitpick: should there be a space before each `-`?

Reply:
Yes, there should be! Corrected it 😃